rfcs/20201202-dnnl-verbose-extension/README.md

```diff
@@ -0,0 +1,40 @@
+# RFC: DNNL_VERBOSE functionality extension
+
+## Motivation
+
+oneDNN users wish to profile the library calls via chrome trace, which is
+popular among different products. In order to have it working properly,
+DNNL_VERBOSE should dump a timestamp of each call, otherwise it's impossible
+to pin a call to a certain point in the timeline. It may be either an absolute
+time value in milliseconds or relative to a certain start point dumped once.
+Both ways will let to profile each call through the whole program execution.
+
+## Proposal
+
+To keep the current behavior intact, the proposal is to introduce a modifier
+which will be enabled by additional environment variable
+`DNNL_VERBOSE_TIMESTAMP`. When set to `0` (the default), it does not affect the
+existing output of DNNL_VERBOSE. When set to `1`, it adds absolute time in
+milliseconds. There are two options where to put this timestamp:
```

Comment:
We may want to add that "the epoch is unspecified". `get_msec()` returns different times on Linux and Windows:
- On Linux: time since the Unix epoch on Linux
- On Windows: time since the last system start

Reply:
It's a good point. I've added a note reflecting the implementation detail. Thanks.